app/frontend/src/views/form/Manage.vue

```diff
@@ -1,141 +1,232 @@
+          </div>
+        </v-expansion-panel-header>
+        <v-expansion-panel-content>
+          <v-form
+            ref="settingsForm"
+            :disabled="formSettingsDisabled"
+            v-model="settingsFormValid"
+            lazy-validation
+          >
+            <FormSettings :disabled="formSettingsDisabled" />
+          </v-form>
+          <v-btn
+            :disabled="formSettingsDisabled"
+            class="mr-5"
+            color="primary"
+            @click="updateSettings"
+          >
+            <span>Update</span>
+          </v-btn>
+          <v-btn
+            :disabled="formSettingsDisabled"
+            outlined
+            @click="cancelSettingsEdit"
+          >
+            <span>Cancel</span>
+          </v-btn>
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
+
+    <v-expansion-panels
+      v-model="versionsPanel"
+      flat
+      class="nrmc-expand-collapse mt-5"
+    >
+      <v-expansion-panel flat>
+        <v-expansion-panel-header>
+          <template v-slot:actions>
+            <v-icon class="icon">$expand</v-icon>
+          </template>
+          <div class="header">
+            <span>Form Design</span>
+            <span>
+              <small>
+                Last Updated:
+                {{ currentVersion.updatedAt | formatDateLong }}
+                <span v-if="currentVersion.updatedBy">
+                  ({{ currentVersion.updatedBy }})
+                </span>
+              </small>
+            </span>
+          </div>
+        </v-expansion-panel-header>
+        <v-expansion-panel-content>
+          <p>
+            <strong>Current Version:</strong>
+            {{ currentVersion.version }}
             <router-link
               :to="{
                 name: 'FormDesigner',
                 query: { f: f, v: currentVersion.id },
               }"
             >
-              <v-btn color="blue" text small>
-                <v-icon class="mr-1">edit</v-icon>
+              <v-btn v-if="canCreateDesign" color="primary" text small>
+                <v-icon class="mr-1 ml-5">edit</v-icon>
                 <span>Edit Current Form</span>
               </v-btn>
             </router-link>
-            <br />
-            <br />
-            <v-btn color="blue" text small>
-              <v-icon class="mr-1">add</v-icon>
-              <span>Add New Version</span>
-            </v-btn>
-          </v-card-text>
-        </v-card>
-        <br />
-        <v-card outlined>
-          <v-card-text>
-            <div class="overline mb-4">Status Workflow</div>
-            <h2>TBD</h2>
-            <v-btn color="blue" text small>
-              <v-icon class="mr-1">edit</v-icon>
-              <span>Modify</span>
-            </v-btn>
-          </v-card-text>
-        </v-card>
-      </v-col>
-    </v-row>
+          </p>
+
+          <BaseInfoCard>
+            Editing this form and saving the changes will create and publish a
```

Comment:
May need to run this by Mina, but we should keep differentiating between Form (Form Settings) and Form Design.  So this should probably say 'Editing this form design'

Reply:
That's a good point. This is temp wording so didn't get it from the mockups. Will change to that.